src/plat/qemu-arm-virt/config.cmake

```diff
@@ -12,7 +12,10 @@ declare_platform(qemu-arm-virt KernelPlatformQEMUArmVirt PLAT_QEMU_ARM_VIRT Kern
 set(qemu_user_top 0xa0000000)
 
 macro(setup_qemu_armv7)
-    cmake_parse_arguments(ARMV7_OPTIONS "ve" "" "")
+    cmake_parse_arguments(ARMV7_OPTIONS "ve" "" "" ${ARGN})
```

Comment:
The general idea of this mechanism was having a way to be able to specify ARM architecture extensions per core. Practically it is a bit overkill for what we currently do. Especially since both Cortex-A7 and A15 support "ve" actually, and these are the only ARMv7 cores supported by QEMU virt. 
Can we we postpone changing this until after this PR is merged, so the original issue gets fixed?

Reply:
Happy to leave that for a separate PR.